ncm-ceph/src/main/perl/Ceph/commands.pm

```diff
@@ -161,8 +161,8 @@ sub ssh_known_keys {
         if (!$lines) {
```

Comment:
It needs to trim the newlines, because of the behaviour of ssh-keygen:

[kwaegema@daenerys ~]$ bla=`` `ssh-keygen -F gastlyf.ugent.be` ``
[kwaegema@daenerys ~]$ echo $bla

[kwaegema@daenerys ~]$